docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,197 @@
+scheduler preempts pods until the pending pod (P) can be scheduled on the node (N).
+In the meantime, scheduler keeps scheduling other pending pods. When one or more victims
+exit or get terminated, scheduler may place other pending pods on the node if the pods are
+ahead of P in the scheduling queue. In such a case, it is likely that
+when all victims exit, pod P won't fit on node N anymore. So, scheduler will have to
+preempt other pods on node N or another node to let P schedule. This scenario may 
+be repeated again for the second and subsequent rounds of preemption and P may not
+get scheduled for a while. This scenario can cause problems in various clusters, but
+is particularly problematic in clusters where many new pods are created all the time.
+
+We will address this problem in beta version of pod preemption. The solution
+we plan to implement is [provided here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/pod-preemption.md#preemption-mechanics).
+
+#### PodDisruptionBudget is not supported
+[Pod Disruption Budget (PDB)](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/)
+allows application owners to limit the number pods of a replicated application that
+are down simultaneously from voluntary disruptions. However, alpha version of
+preemption does not respect PDB when choosing preemption victims.
+We plan to add PDB support in beta, but even in beta respecting PDB will be best
+effort. Scheduler will try to find victims whose
+PDB won't be violated by preemption, but if no such victims are found, preemption
+will still happen and lower priority pods will be removed despite their PDBs 
+being violated.
+
+#### Inter-Pod Affinity on Lower Priority Pods
+The current implementation of preemption considers a node for preemption only when
+the answer to this question is positive: "If all the pods with lower priority than
+the pending pod are removed from the node, can the pending pod be scheduled on
+the node?"
+(Note that preemption does not always remove all lower-priority pods, e.g. if the 
+pending pod can be scheduled by removing fewer than all lower-priority pods, but this
+test must always pass for preemption to be considered on a node.)
+
+If the answer is no, that node will not be considered for preemption. If the pending
+pod has inter-pod affinity to one or more of those lower priority pods on the node, the
+inter-pod affinity rule cannot be satisfied in the absence of the lower priority
+pods and scheduler will find the pending pod infeasible on the node. As a result,
+it will not try to preempt any pods on that node.
+Scheduler will try to find other nodes for preemption and could possibly find another
+one, but there is no guarantee that such a node will be found.
+
+We may address this issue in future versions, but we don't have a clear plan and cannot
+promise that it will be fixed in Beta or GA. Part
```

Comment:
I think @wojtek-t would feel better if after "GA" you say "(i.e. we will not consider it a blocker for Beta or GA)."

Reply:
Sure. Done :)